client/.eslintrc.json

```diff
@@ -2,7 +2,8 @@
     "env": {
         "commonjs": true,
         "es2021": true,
-        "node": true
+        "node": true,
+        "browser": true
```

Comment:
In `client/src/components/landing-pages/landingPageComponents/landingPageLayouts/LandingHeader.jsx` (line 31 ++), variable `window` and `browser` are used, adding this line of code removes the warning message.

Reply:
Thanks for this, I didn't know you could do it like this. I was just suppressing the rules line by line - could remove those comments as well?

Reply:
Which comment are you refering to? If you're saying the comment in `line34`: 
```
33     const navbar = document.getElementById('navbar');
34     window.onscroll = function () { // eslint-disable-line func-names
35        const maxScroll = document.body.clientHeight - window.innerHeight;
36        const currentScrollPos = window.pageYOffset;
```
I think we still need to keep the comment to suppress unnamed function, unless we give the function a name?

Reply:
if you do a search for `no-undef`, these come up:
![image](https://user-images.githubusercontent.com/40584352/132679199-c826e812-2ee9-4351-9e39-f3b4e06b9be3.png)
for localstorage, window and document

Reply:
okie I'll change them